### PR TITLE
created payment view

### PR DESCRIPTION
**Pull request by HubDub**
## Pull Request Description

Include the applicable information from the list below here 

this view is a form the user will fill in to create a new payment type. 
## Steps to Test

review the code on github for any errors. we cannot access this section of the site on the dotnet run just yet. 
## Link to Feature Ticket
#45

### CODE REVIEW

**Review by csmalley89 — COMMENTED** [review]
👍

**Review by Jgregnier — COMMENTED** [review]
Looks good to me

**Review by JamsDuke — COMMENTED** [review]
Looks good

**Review by thrashMoneyVillainare — COMMENTED** [review]
lgtm

File: Views/Customers/Payment.cshtml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+@model BangazonWeb.ViewModels.PaymentTypeViewModel
+
+@{
+    ViewData["Title"] = "New Payment Type";
+}
+
+<h1>Add a New Payment Type</h1>
+
+<form asp-action="Payment">
```
Comment (delaineackerman) [inline Views/Customers/Payment.cshtml:9]: Make sure you add method="post" and asp-controller="Customer"

**Review by Jgregnier — APPROVED** [review]

**Pull request merged**